src/Symfony/Component/AssetMapper/MappedAsset.php

```diff
@@ -24,7 +24,7 @@ final class MappedAsset
     public readonly string $publicPath;
     public readonly string $publicPathWithoutDigest;
     public readonly string $publicExtension;
-    public readonly string $content;
+    public readonly ?string $content;
```

Comment:
It might deserve some PHPdoc above:

```php
/**
 * The final content of this asset, or null if the source file's content should be used.
 */
```